HomeTask7.6/HomeTask7.6/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace HomeTask7._6
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("В каких разрядах вы хотите найти простые числа?");
+            int NumberOfCategory = int.Parse(Console.ReadLine());
+            int Category = 10;
+            if (NumberOfCategory > 1)
+                for (int i = 1; i <= NumberOfCategory - 1; i++)
+                {
+                    Category = Category * 10;
+                }
```

Comment:
Локальные переменные принято именовать с маленькой буквы.
И тут можно было обойтись без цикла.
```suggestion
            int numberOfCategory = int.Parse(Console.ReadLine());
            int category = int.Parse("1" + new string('0', NumberOfCategory));
```